src/main/java/dev/lambdaurora/aurorasdeco/block/BlackboardBlock.java

```diff
@@ -373,9 +374,10 @@ public ItemStack getPickStack(BlockView world, BlockPos pos, BlockState state) {
 		var stack = super.getPickStack(world, pos, state);
 		var blackboard = this.getBlackboardEntity(world, pos);
 		if (blackboard != null) {
-			var nbt = blackboard.writeBlackBoardNbt(new NbtCompound());
+			var nbt = new NbtCompound();
+			blackboard.writeBlackBoardNbt(nbt);
```

Comment:
Not a fan of this change, the one liner has actually better readability imo, since it emphasize what the local holds.

Reply:
Makes sense, reverted this one (and two similar ones in `BlackboardBlock.onBreak` and `HoneycombItemMixin.onBeforeReplace`).